src/main/groovy/com/github/dispader/manifesto/ManifestoPluginExtension.groovy

```diff
@@ -4,7 +4,9 @@ import org.gradle.api.Project
 
 class ManifestoPluginExtension {
 
-    String vendor, vendor_id, url
+    String vendor = ""
+    String vendor_id = ""
+    String url = ""
```

Comment:
I can see defaulting the values here.  Could we make this non-interpolated (single-quoted) `String`s?  Here they're (double-quoted) `GStrings`, which is popping a Codenarc violation.